source/Mission.cpp

```diff
@@ -588,6 +608,45 @@ int Mission::CargoSize() const
 
 
 
+const Outfit *Mission::GetOutfit() const
+{
+	return outfit;
+}
+
+
+
+int Mission::OutfitUnits() const
+{
+	return outfitUnits;
+}
+
+
+
+std::string Mission::OutfitName() const
+{
+	if(!outfit)
+		return "";
+	return outfit->TrueName();
```

Comment:
I feel like this shouldnt even  be called if there is no outfit
But its prob fine, do use a ternary operator though:
`return outfit ? outfit->TrueName() : ""`